odoo/addons/base/models/res_partner.py

```diff
@@ -451,21 +451,22 @@ def _fields_sync(self, values):
         self._children_sync(values)
 
     def _children_sync(self, values):
-        if not self.child_ids:
+        child_ids = self.with_context(active_test=False).search([('parent_id', '=', self.id)])
```

Comment:
by convention child_ids should be a list of ID. here it is a recordset
child_ids --> childs
@rco-odoo